support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouSupportReminder.tsx

```diff
@@ -127,6 +128,22 @@ const getDefaultReminderChoices = (): ReminderChoice[] => [
 	getOneOffReminderChoiceWithDefaultLabel(9),
 ];
 
+const GIVING_TUESDAY_CHOICE: ReminderChoice = {
+	type: 'ONE_OFF',
```

Comment:
This diff in this file is really big because of a formatting change. I did that as a commit first, so it's probably easiest to review by just looking at the second commit in this branch